### ISSUE TITLE
Multiple "Use template" buttons are not uniquely identified for screen reader users.:A11y_Template Gallery_Browse Template_ScreenReader

### ISSUE THREAD

**Issue by Mana-swi** [microsoft/vscode-azurefunctions#5172]
### GitHub Tags
#A11yTCS; #SH_TemplateGallery_Win32_Aug2026; #Azure Tools for VS Code; #DesktopApp; #Win32; #Win11; #A11ySev2; #WCAG2.4.6; #Headings and Labels; #a11ydb-label-unclear; #A11yWCAG;

### Issue Subcategory
Label text does not describe its control

### User Experience
Users cannot determine the purpose of a form field or control from its label alone. They must guess what data to enter or rely on surrounding context clues.

### Repro Steps
Open VS Code > Install azure functions extension -> Open command pallet -> Create new project -> Select Browse Template Gallery -> Search for any template.

### Actual Result
Each control is announced only as "Use template, button" when navigating directly between buttons. The associated template name is not included, so the buttons cannot be distinguished without leaving button navigation and reviewing surrounding content.

### Expected Result
The Use template button labels should clearly describe the purpose of each control and identify the associated template.

### How to Fix
Rewrite the label text so that it provides an unambiguous description of its purpose. A clear label may simply be one or two words - so long as those words are meaningful to the purpose of the control.

### Environment details
Application Name: Template Gallery (Azure Tools for VS Code)
VS code Version: 1.136.1 (user setup)
Electron: 42.10.0
ElectronBuildId: 15109253
Chromium: 148.0.7778.280
Node.js: 24.18.1
V8: 14.8.178.38-electron.0
OS: Windows_NT x64 10.0.26680

### Note
Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification.
"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Attachment

<img width="853" height="480" alt="Image" src="https://github.com/user-attachments/assets/840128b3-be95-4cd5-a92a-7293a1ff9be3" />

UIA tree:
<img width="1364" height="740" alt="Image" src="https://github.com/user-attachments/assets/fe38d7bb-d775-415a-9168-7a14638960b4" />
